modules/gathering-data-specific-features.adoc

```diff
@@ -56,7 +56,18 @@ ifndef::openshift-rosa,openshift-dedicated[]
 |Data collection for {sandboxed-containers-first}.
 
 |`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
-|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
+|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).
+
+Use this image if your NHC Operator version is *less than 0.9.0*. 
+
+For more information, see the "Gathering data" section for the specific Operator in link:https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).
```

Comment:
```suggestion
For more information, see the "Gathering data" section for the specific Operator in https://docs.redhat.com/en/documentation/workload_availability_for_red_hat_openshift/latest/html/remediation_fencing_and_maintenance/index[Remediation, fencing, and maintenance] (Workload Availability for Red Hat OpenShift documentation).
```
the corrected link for that section of the docs

Reply:
Thanks @ShaunaDiaz for your feedback. It is really helpful. I have applied your suggestions.